### DISCUSSION TITLE
PostgresSQL support

### DISCUSSION THREAD

**Discussion by seddik** [maxtoroq/DbExtensions#67] [Q&A] [answered]

**Comment by maxtoroq** [comment]
Some users have used it successfully with PostgresSQL.

**Comment by seddik** [comment]
@maxtoroq i didn't see the provider name in the sourcecode, did they use theirs own implementation, or it is possible to use mysql as a provider ??!!

**Comment by maxtoroq** [comment, marked as answer]
DbExtensions can work without any specific provider details because it uses DbCommandBuilder to get that stuff.

When I released v6.0, .NET Core didn't have SqlCommandBuilder, so I introduced the concept of known providers, for SqlServer, MySQL, etc. I also added config properties to DatabaseConfiguration, so even if the provider does not implement the needed methods on DbCommandBuilder, you can set those properties yourself.

**Comment by seddik** [comment]
thank you so much, I've already tried V5.0 on a large scale application, and its working like charm 😄
